### PR TITLE
Simp_le upgrade

### PR DESCRIPTION
**Pull request by sagepe**
Updates for compatibility with simp_le >= 0.15.0 and Python 3.

### CODE REVIEW

**Comment by sagepe** [comment]
I'll turn this into a full PR once I've worked out why we've been getting `CA did not offer http-01-only challenge combo.` errors from the live endpoint (but not the staging one).

**Comment by sagepe** [comment]
Renewals have been functioning normally, and given the error noted above happened with both versions of this code I don't think it's to do with this PR.

File: renew.py [hunk 1]
```diff
@@ -179,12 +180,12 @@ def get_cert_data():
         for crt in sorted(glob.glob('/etc/ssl/mysociety/certs/*.crt')):
             with open(crt) as f:
                 cert = OpenSSL.crypto.load_certificate(OpenSSL.crypto.FILETYPE_PEM, f.read())
-            expiry = cert.get_notAfter()
+            expiry = cert.get_notAfter().decode('utf-8')
```
Comment (dracos) [inline renew.py]: `ascii`, maybe? given this should always be an ascii string and presumably you'd want it to error if it somehow got unicode in it :)

File: utils.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def _get_vhost_domains(self, vhost_name, data):
     def _parse_vhosts_pl_section(self, vhosts_pl_path, section):
         return json.loads(subprocess.check_output([
             'perl', '-e', 'use JSON; require "' + vhosts_pl_path + '"; print encode_json($' + section + ');'
-        ]))
+        ]).decode('utf-8'))
```
Comment (dracos) [inline utils.py:73]: I realise we're on 3.5, just to note that from python 3.6 json.loads() does accept bytes here.

**Review by dracos — APPROVED** [review]
Wasn't sure if this was now to be python3 only - if it were, you could e.g. pass encoding to the check_output call I think, rather than doing it after. No matter.

**Pull request merged**